### PR TITLE
Get started with building responsive webpages, Project 2 Technigo

### PR DESCRIPTION
**Pull request by Kras053**

### CODE REVIEW

**Comment by Kras053** [comment]
Deploy:
https://cocky-northcutt-6390f0.netlify.app/

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,192 @@
...
+display:grid;
+grid-template-columns: 1fr; 
+margin-top: 50px;
+margin-left: 25px;
+margin-right: 25px;
+margin-bottom: 50px;
+}
+
+.other-news img {
+width: 300px;
```
Comment (mistscale) [inline code/style.css:148]: Suggestion to set the img width to 100% to make them fit better within the border

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,192 @@
...
+.icon2 {
+background-color: white;
+  }
+}
+  
+@media (min-width: 668px) {
+.icon3 {
+background-color: white;
+  }
+}
```
Comment (mistscale) [inline code/style.css:92]: Maybe all media queries with same min-width could be written under the same media query to make it more easy to read

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,192 @@
...
+
+.other-news a {
+text-decoration: none;
+color: black;
+border-style: solid;
+border-color: grey;
+border-width: thin;
+padding: 15px;
+}
```
Comment (mistscale) [inline code/style.css:158]: Beautiful styling with the thin border!

File: code/index.html [hunk 4]
```diff
@@ -4,19 +4,105 @@
...
+    <hr class="solid-line">
+
+    <div id="articles"></div>
+
+    <section class="other-news">
+    <a href="#">
+    <img src="https://images.unsplash.com/photo-1612490689975-c062b1bdcec7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80"
+      alt="a paperheart taped on a matchstick"/>
+    <span><h3>Valentine's Day special: cute and fun crafting.</h3></span>
+    <div class="hidden-hover">Because who doesn't love presents? 🎁🧡</div></a>
```
Comment (mistscale) [inline code/index.html:63]: Very cool effect with the hover in the news! I really like the orange styling of them as well.

**Review by mistscale — COMMENTED** [review]
Good job with the project Louisa! 🤩
Very fun topic and the code structure was easy to follow. The site works well with the media queries and it was interesting to read how you solved the project using grid layout.

**Pull request closed without merging**